src/status_im2/contexts/profile/profiles/views_v2.cljs

```diff
@@ -0,0 +1,123 @@
+(ns status-im2.contexts.profile.profiles.views-v2
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
+            [quo.design-system.colors :as colors]
+            [re-frame.core :as re-frame]
+            [reagent.core :as reagent]
+            [status-im.ui.components.copyable-text :as copyable-text]
```

Comment:
same for the status-im code, this is all old and needs to be removed on our side. Yet for this pr you should also not need it. This code should exist elsewhere, it's quite tricky to see what you are using here so please reach out for individual questions. 

in the case of ` [status-im.ui.components.react :as react]`
it's preffered to use [react-native.core :as rn]. you'll see a lot `rn/view` etc in the codebase 👍

Reply:
Copied from the original file and changed some as current implementation, so some `required` I still don't understand clearly. Deleted most of theme.

Reply:
sure, np. if there is some expectation or something that is unclear it's best to discuss first. I suggest reaching out over discord as it will be faster to discuss there. 👍